tests/model_selection/test_incremental.py

```diff
@@ -200,7 +200,9 @@ def test_search_basic(c, s, a, b):
 
     params = {"alpha": np.logspace(-2, 2, 100), "l1_ratio": np.linspace(0.01, 1, 200)}
 
-    search = IncrementalSearchCV(model, params, n_initial_parameters=20, max_iter=10)
+    search = IncrementalSearchCV(
+        model, params, n_initial_parameters=20, max_iter=10, decay_rate=0
+    )
```

Comment:
Why this change?  Why not also engage decay rate here?

Reply:
Because for this dataset and model, I had to make a change to the tests (same change as in https://github.com/dask/dask-ml/pull/370#issuecomment-428788259):

``` diff
- assert (
- search.cv_results_["test_score"][search.best_index_]
-         >= search.cv_results_["test_score"]
- ).all()
- search.cv_results_["rank_test_score"][search.best_index_] == 1
+ assert (
+ search.cv_results_["test_score"][search.best_index_]
+          >= np.percentile(search.cv_results_["test_score"], 90)
+ )
+ search.cv_results_["rank_test_score"][search.best_index_] <= 3
```

I interpret this change as meaning "random searches work best", which makes sense because the model/dataset are simple and not performant. Plus, this test is aimed at testing basic elements of `IncrementalSearchCV` but also relies on the adaptive algorithm working well.

Reply:
That is, without `decay_rate=0`, this search wouldn't return the highest scoring model (with this small dataset and simple model, the search is too adaptive).